src/main/daml/Daml/Finance/Interface/Data/Numeric/Observation.daml

```diff
@@ -0,0 +1,53 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Why do we introduce this new interface here?

Reply:
Ah, it's the interface for our specific implementation of `NumericObservation` that has the `observations` list so that we can have a factory for it. Make sense, but agree it feels quite boilerplate-y. 🤷‍♂️

Reply:
This is not strictly required, but at the moment we do not have an interface that matches 1 to 1 with the `Observation` template. 

The `NumericObservable` interface is meant to be implemented by many templates and gives access to the `Observe` choice, but it hides the internal data structure of the template.

In order to display an `Observation` in a UI, or to manipulate observations, one would need to abstract it with an interface that preserves the template's data structure (such that no information is lost). Hence I added the `Observation` interface.

Let me know what you think, we could also have the factory return a `NumericObservable` directly if you think that makes more sense.

Reply:
I think that from a client perspective it is easier to have a custom implementation of `Observation`, in order to avoid having to deal with factories and the additional interfaces. 

However, this would require a change in the target architecture for a Daml Finance app, where we would say
- use factories for these packages
- copy over the implementation for these other packages

which could be a bit confusing for a customer getting started with the library.

Reply:
I think it makes sense as it is 👍 Valid point about a customer wanting to use their own observation templates, but I think it's good if we provide a "general purpose" one out of the box (together with the associated interfaces).